tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -149,3 +154,4 @@ class Production(CloudGov):
         'urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-prod'
     )
     ENABLE_DEVELOPER_GROUP = False
+    MIDDLEWARE = ('tdpservice.middleware.SessionMiddleware', *Common.MIDDLEWARE)
```

Comment:
This line is definitely needed as it should only be added in prod.

I am almost certain `tdpservice.middleware.SessionMiddleware` will break dev and staging if it is in the middleware.

Reply:
And I am certain it will break it :)

The tdpservice middleware has lines that directly modify how sessions are responded to including headers which hardcode the ACF domains. We want everything non-prod to use django's built-in session middleware.